components/backup-plugins/Gopkg.toml

```diff
@@ -45,13 +56,29 @@ required = [
   name = "github.com/kubernetes-sigs/service-catalog"
   version = "=v0.2.1"
 
-[[constraint]]
-  name = "k8s.io/api"
-  version = "kubernetes-1.14.0"
+[[override]]
+  name = "github.com/knative/pkg"
+  branch = "release-0.8"
 
 [[constraint]]
+  name = "github.com/knative/eventing"
+  version = "v0.8.0"
+
+[[override]]
+  name = "contrib.go.opencensus.io/exporter/stackdriver"
+  version = "=v0.12.2"
+
+[[override]]
+  name = "k8s.io/client-go"
+  version = "kubernetes-1.12.9"
+
+[[override]]
   name = "k8s.io/apimachinery"
-  version = "kubernetes-1.14.0"
+  version = "kubernetes-1.12.9"
```

Comment:
why you are downgrading k8s?

Reply:
Because knative is requiring strictly `1.12.9` and Velero plugins don't require features of the `1.14.0`

Reply:
feels strange as kyma requires 1.14, but ok